resources/js/features/events/components/AchievementSet/AchievementDateMeta/AchievementDateMeta.tsx

```diff
@@ -36,10 +38,12 @@ export const AchievementDateMeta: FC<AchievementDateMetaProps> = ({
   let isUpcoming = false;
   if (eventAchievement && event?.state !== 'evergreen' && activeFrom && activeThrough) {
     const now = dayjs.utc();
+    const activeFrom = dayjs.utc(eventAchievement.activeFrom);
+    const activeUntil = dayjs.utc(eventAchievement.activeUntil);
 
-    isActive = dayjs.utc(activeFrom).isBefore(now) && dayjs.utc(activeThrough).isAfter(now);
-    isExpired = dayjs.utc(activeThrough).isBefore(now);
-    isUpcoming = dayjs.utc(activeFrom).isAfter(now);
+    isActive = activeFrom.isSameOrBefore(now) && now.isBefore(activeUntil);
+    isExpired = activeUntil.isBefore(now);
```

Comment:
Technically, this should be `isSameOrBefore`, but I'm not sure if anyone's going to be looking right at crossover.

Reply:
Updated in latest.